src/basic/MurmurHash2.c

```diff
@@ -30,61 +30,61 @@ _Pragma("GCC diagnostic ignored \"-Wimplicit-fallthrough\"")
 
 // Other compilers
 
-#else        // defined(_MSC_VER)
+#else // defined(_MSC_VER)
```

Comment:
hmm, MurmurHash2.c is imported from another source base mostly, which is why we kept its indenting in place like it is...

We could probably reindent it, if its done automatically, in particular, but I'd probably not do it if it's easy to exclude

Reply:
I can exclude the file...